app/src/main/res/layout/activity_onboarding_web_view.xml

```diff
@@ -18,5 +19,57 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content" />
 
+    <android.support.design.widget.AppBarLayout
+        android:id="@+id/app_bar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:theme="@style/AppTheme.AppBarOverlay"
+        android:background="@color/transparent"
+        android:layout_marginTop="7dp"
+        android:fitsSystemWindows="false"
+        android:layout_alignTop="@+id/activity_onboarding_web_view"
+        app:elevation="0dp">
+
+        <android.support.design.widget.CollapsingToolbarLayout
```

Comment:
We were trying something out in order to match the designs. We'll change this to a normal toolbar in https://github.com/praekeltfoundation/gem-bbb-indo/pull/163